sdk/storage/Azure.Storage.Files.Shares/perf/Azure.Storage.Files.Shares.Perf/Scenarios/DownloadFile.cs

```diff
@@ -0,0 +1,151 @@
+        /// </summary>
+        public override async Task SetupAsync()
+        {
+            await base.SetupAsync();
+
+            // Upload a file to the share storage. The test will use 'ShareClient' to download this file.
+            (FilesShareClient, DirectoryClientName, FileClientName) = await _fileUploader.UploadARandomFileAsync();
+        }
+
+        /// <summary>
+        /// Performs the tasks needed to clean up the environment for an instance of the test scenario.
+        /// When multiple instances are run in parallel, cleanup will be run once for each after execution has completed.
+        /// </summary>
+        public override async Task CleanupAsync()
+        {
+            await base.CleanupAsync();
+
+            try
+            {
+                await _fileUploader.CleanupAsync();
+                File.Delete(DownloadFilePath);
+            }
+            catch
+            {
+            }
+        }
+
+        /// <summary>
+        /// Executes the performance test scenario synchronously.
+        /// </summary>
+        /// <param name="cancellationToken">The token used to signal when cancellation is requested.</param>
+        public override void Run(CancellationToken cancellationToken)
+        {
+            ShareDirectoryClient directoryClient = FilesShareClient.GetDirectoryClient(DirectoryClientName);
+            ShareFileClient fileClient = directoryClient.GetFileClient(FileClientName);
+
+            Models.ShareFileDownloadInfo fileDownloadInfo = fileClient.Download(cancellationToken: cancellationToken);
+
+            // We only want to measure the download operation. So, skip the part of creating a local file with the downloaded stream.
+            //using (FileStream stream = File.OpenWrite(DownloadFilePath))
+            //{
+            //    fileDownloadInfo.Content.CopyTo(stream);
+            //}
+
+#if DEBUG
+            Console.WriteLine($"Downloaded file from {fileClient.Path}. Length: {fileDownloadInfo.ContentLength}");
+#endif
+        }
+
+        /// <summary>
+        /// Executes the performance test scenario asynchronously.
+        /// </summary>
+        /// <param name="cancellationToken">The token used to signal when cancellation is requested.</param>
+        public override async Task RunAsync(CancellationToken cancellationToken)
+        {
+            ShareDirectoryClient directoryClient = FilesShareClient.GetDirectoryClient(DirectoryClientName);
+            ShareFileClient fileClient = directoryClient.GetFileClient(FileClientName);
+
+            Models.ShareFileDownloadInfo fileDownloadInfo = await fileClient.DownloadAsync(cancellationToken: cancellationToken);
+
+            // We only want to measure the download operation. So, skip the part of creating a local file with the downloaded stream.
+            //using (FileStream stream = File.OpenWrite(DownloadFilePath))
```

Comment:
I don't thin the data is being downloaded if we don't read the stream that's in fileDownloadInfo.Content

Reply:
Hmm, ok. Let me see if I can find some definitive doc for that.
I am checking the `ContentLength` to verify the download.

Reply:
Can you please post the results you've got after running the benchmark?

Reply:
I am getting ~10.25 ops/s in no-parallelism runs.

Reply:
Interesting.   That hadn't occurred to me.   I wonder if the same is for the `Value` in the Data Lake response.

Reply:
Does it change when you vary the file size?

Reply:
I am currently getting 
Sync mode, no-parallelism
1KB -    8.70 ops/s
1MB -   7.94 ops/s
10MB - 6.78 ops/s

The numbers look pretty stable. I have not analyzed them in detail.
I will recheck them with when the test is reading the downloaded stream.

Reply:
I can verify the data isn't being downloaded when you don't read the stream. This function will move on the moment the SDK gets the response header and turns that into a BlobDownloadInfo result.

Reply:
Awesome, thanks for the reply. I will update the test.